src/Profiles/LEV/Display/ANTPLUS_ProfileLevDisplay.h

```diff
@@ -0,0 +1,53 @@
+#ifndef ANTPLUS_PROFILELEVDISPLAY_h
+#define ANTPLUS_PROFILELEVDISPLAY_h
+
+#include <BaseClasses/ANTPLUS_BaseSlaveProfile.h>
+#include <Profiles/LEV/DataPages/ANTPLUS_ProfileLevDataPages.h>
+#include <CommonDataPages/ANTPLUS_CommonDataPages.h>
+
+#include "ANT.h"
+
+class ProfileLevDisplay : public BaseSlaveProfile {
+public:
+    ProfileLevDisplay();
+    ProfileLevDisplay(uint16_t deviceNumber);
+    ProfileLevDisplay(uint16_t deviceNumber, uint8_t transmissionType);
+
+	void onLevSpeedSysinfo1(void(*func)(LevSpeedSysinfo1&, uintptr_t), uintptr_t data = 0) { _onLevSpeedSysinfo1.set(func, data); }
+	void onLevSpeedDist1(void(*func)(LevSpeedDist1&, uintptr_t), uintptr_t data = 0) { _onLevSpeedDist1.set(func, data); }
+	void onLevSpeedDist2(void(*func)(LevSpeedDist2&, uintptr_t), uintptr_t data = 0) { _onLevSpeedDist2.set(func, data); }
+	void onLevSpeedSysinfo2(void(*func)(LevSpeedSysinfo2&, uintptr_t), uintptr_t data = 0) { _onLevSpeedSysinfo2.set(func, data); }
+	void onLevBatteryInfo(void(*func)(LevBatteryInfo&, uintptr_t), uintptr_t data = 0) { _onLevBatteryInfo.set(func, data); }
+	void onLevCaps(void(*func)(LevCaps&, uintptr_t), uintptr_t data = 0) { _onLevCaps.set(func, data); }
```

Comment:
Also indentation

Reply:
Names refactored, Indentation unclear (changed my editor from tabs to spaces)

Reply:
Ah, its probably stuck on tabs, whitespace is awful to fix, I can send a pull request onto your branch with fixes, usually sublime text is the easiest way to convert it since it has built in utilities.

Reply:
Done and checked. Visual Studio editor adjusted to spaces instead tabs